src/utils/UpdateNotification.js

```diff
@@ -31,6 +31,7 @@ define(function (require, exports, module) {
     var Dialogs              = require("widgets/Dialogs"),
```

Comment:
Can all the changes made here be moved to the extension?

Reply:
The changes made here for AutoUpdate feature heavily use the update information like version info, URLs etc., which is maintained in this file. Hence, the changes should not be moved to the extension.

Reply:
@mbhavi One of the main reasons, we move most of the new code to an extension is to leverage pluggability model. So imagine, `AutoUpdate` extension is removed from the list of default extensions, the core should function fine and preferably with other functionality, which in this case is just open the `brackets.io` page. You need to follow something like what @swmitra has done with https://github.com/adobe/brackets/pull/14140. Have a default implementation, which an extension can override and provide implementation on how to get the latest update.